<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrarAdminDialog</class>
 <widget class="QDialog" name="RegistrarAdminDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Usuario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usuarioLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Contraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="contrasenyaLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Repite contraseña</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="repiteContrasenyaLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="adminCheckBox">
     <property name="text">
      <string>Agregar como administrador</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
